docs/integrations/airflow/airflow.md

```diff
@@ -22,50 +22,17 @@ The DAG metadata collected can answer questions like:
 
 ## How can I use this integration?
 
-#### PREREQUISITES
+To instrument your Airflow instance with OpenLineage, follow [these instructions](usage.md).
 
-To use the OpenLineage Airflow integration, you'll need a running Airflow instance. You'll also need an OpenLineage compatible [backend](https://github.com/OpenLineage/OpenLineage#scope).
+## How to add lineage coverage for more operators?
 
-#### INSTALLATION
+OpenLineage provides a set of `extractors` to extract lineage from operators. 
```

Comment:
Suggest "a set of extractors that extract lineage from operators" instead of "to", for readability